modules/drivers/bigquery-cloud-sdk/test/metabase/driver/bigquery_cloud_sdk_test.clj

```diff
@@ -419,6 +419,105 @@
+                                                  :table_id      (mt/id :cf_product)
+                                                  :dataset_query (mt/mbql-query cf_product)}
+              :model/Dashboard     dashboard     {:parameters [{:name "Category"
+                                                                :slug "category"
+                                                                :id   "_CATEGORY_"
+                                                                :type :string/=}
+                                                               {:name "Product Name"
+                                                                :slug "Product name"
+                                                                :id   "_NAME_"
+                                                                :type :string/=}]}
+              :model/DashboardCard _dashcard     {:card_id            (:id card-category)
+                                                  :dashboard_id       (:id dashboard)
+                                                  :parameter_mappings [{:parameter_id "_CATEGORY_"
+                                                                        :card_id      (:id card-category)
+                                                                        :target       [:dimension (mt/$ids $cf_category.category)]}]}
+              :model/DashboardCard _dashcard     {:card_id            (:id card-product)
+                                                  :dashboard_id       (:id dashboard)
+                                                  :parameter_mappings [{:parameter_id "_NAME_"
+                                                                        :card_id      (:id card-product)
+                                                                        :target       [:dimension (mt/$ids $cf_product.name)]}]}]
+
+             (testing "chained filter works"
+               (is (= {:has_more_values false
+                       :values          [["Americano"] ["Cold brew"]]}
+                      (mt/user-http-request :crowberto :get 200 (format "/dashboard/%d/params/%s/values?%s=%s"
+                                                                        (:id dashboard) "_NAME_" "_CATEGORY_" "coffee")))))
+             (testing "getting values works"
+               (is (= {:has_more_values false
+                       :values          [["Americano"] ["Cold brew"] ["Herbal"] ["Oolong"]]}
+                      (mt/user-http-request :crowberto :get 200 (format "/dashboard/%d/params/%s/values" (:id dashboard) "_NAME_")))))
+             (testing "searching values works"
+               (is (= {:has_more_values false
+                       :values          [["Oolong"]]}
+                      (mt/user-http-request :crowberto :get 200 (format "/dashboard/%d/params/%s/search/oo" (:id dashboard) "_NAME_"))))))
+
+           (finally
+            (doseq [table-name [product-table-name category-table-name]]
```

Comment:
```suggestion
             (doseq [table-name [product-table-name category-table-name]]
```

Reply:
fixed in 5fbe2e50a90f2217d92b646057feb89ef4435c83, and also fixed my config